src/Features/LanguageServer/Protocol/Handler/Definitions/GoToDefinitionHandler.cs

```diff
@@ -18,7 +18,7 @@ public GoToDefinitionHandler(IMetadataAsSourceFileService metadataAsSourceFileSe
         }
 
         public async Task<object> HandleRequestAsync(Solution solution, LSP.TextDocumentPositionParams request,
-            LSP.ClientCapabilities clientCapabilities, CancellationToken cancellationToken, bool keepThreadContext = false)
+            LSP.ClientCapabilities clientCapabilities, CancellationToken cancellationToken, bool keepThreadContext = true)
```

Comment:
please make this non-optional.  It means we actually have to go examine all callers to make sure they're doing the right thing.

Reply:
deleted entirely since I don't need it anymore.  Will address UI issues before moving to this layer.